packages/hosted-form-v2/src/hosted-form-factory.ts

```diff
@@ -23,6 +23,7 @@ export default class HostedFormFactory {
                 new HostedField(
                     type,
                     fieldOptions.containerId,
+                    fieldOptions.orderId,
```

Comment:
Considering `orderId` is identical for all fields, it's better to pass it at the `options` level rather than `fieldOptions` level so the caller doesn't need to repeat.